### DISCUSSION TITLE
TableBlock conflict with BSD licening? Options?

### DISCUSSION THREAD

**Discussion by KingGeorgThe3rd** [wagtail/wagtail#11497] [Q&A] [answered]
Hello everyone. I recently started looking into Wagtail to hopfully get a new company page going ...at some point. So far I am loving what you did here.

Today I looked into how I could make tables easily accessible to any future editor. I noticed that there is the  [TableBlock](https://docs.wagtail.org/en/stable/reference/contrib/table_block.html ) for stream field which sounds promising. Unfortunately I noticed that the software used in the background for this is not for comerical use unless paid for.

I was wondering how this is not in conflict with the BSD licens of Wagtail or at least not mentioned on the documentation. And secondly if there is an alternative how to best make tables usable for editors. I was thinking orderables ...but it seems a bit overboard to me for a simple html table...

Any advice would be appriciated. Thanks

**Comment by zerolab** [comment, marked as answer]
handsontable.js switched to using a commercial license in version 7 (and that applies to versions >= 7). We are still on version 6 for that reason, so there should not be any license issue.

**Reply by KingGeorgThe3rd** [reply]
Thank you so much for pointing this out to me!

**Reply by gasman** [reply]
Thanks for raising this! This has become a bit of an FAQ, so I've now opened https://github.com/wagtail/wagtail/pull/11498 to update the docs to clarify this.
